### PR TITLE
Change newsjob to use Scaleformtext and not Drawtext

### PR DESCRIPTION
**Pull request by mrdiamonddirt**
heading "BREAKING NEWS" done in scaleform added title and subtitle

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes]
- Does your code fit the style guidelines? [yes]
- Does your PR fit the contribution guidelines? [yes]

### CODE REVIEW

**Comment by r0adra93** [comment]
Rowdog please update to latest version so I can go ahead and test this.
-R0adRa93

**Comment by r0adra93** [comment]
Rowdog please update your code so I can test it.

**Comment by r0adra93** [comment]
Come on rowdog update this branch so I can test it!

**Review by r0adra93 — APPROVED** [review]
LGTM

File: locales/es.lua [hunk 1]
```diff
@@ -10,7 +10,9 @@ local Translations = {
         helicopters = "[E] - Helicopteros",
         enter = "[E] - Entrar",
         go_outside = "[E] - Salir",
-        breaking_news= "¡NOTICIA DE ULTIMA HORA!"
+        breaking_news= "¡NOTICIA DE ULTIMA HORA!",
```
Comment (S33G) [inline locales/es.lua]: Missing spaces before `=`

File: locales/en.lua [hunk 2]
```diff
@@ -10,7 +10,9 @@ local Translations = {
         helicopters = "~g~E~w~ - Helicopters",
         enter = "~g~E~w~ - Enter",
         go_outside = "~g~E~w~ - Go outside",
-        breaking_news= "BREAKING NEWS"
+        breaking_news= "BREAKING NEWS",
```
Comment (S33G) [inline locales/en.lua]: Missing space/s before =

**Review by S33G — APPROVED** [review]
LGTM

**Pull request merged**